qtpy/QtWebEngine.py

```diff
@@ -1,19 +1,27 @@
-#
+# -----------------------------------------------------------------------------
 # Copyright © 2014-2015 Colin Duquesnoy
 # Copyright © 2009- The Spyder development Team
 #
 # Licensed under the terms of the MIT License
 # (see LICENSE.txt for details)
+# -----------------------------------------------------------------------------
 
-"""
-Provides QtWebEngine classes and functions.
-"""
+"""Provides QtWebEngine classes and functions."""
 
-from . import PYQT5, PYSIDE6
+from . import PYQT5, PYQT6, PYSIDE2, PYSIDE6, PythonQtError, QtModuleNotInstalledError
 
 if PYQT5:
-    from PyQt5.QtWebEngine import *
+    try:
+        from PyQt5.QtWebEngine import *
+    except ModuleNotFoundError as error:
+        raise QtModuleNotInstalledError(
+            name='QtWebEngine', missing_package='PyQtWebEngine'
+        ) from error
+elif PYQT6:
+    raise PythonQtError('QtWebEngine does not exist in Qt6')
```

Comment:
I think this exist for PyQt6: https://pypi.org/project/PyQt6-WebEngine/ but as with PyQt5 >= 5.12 in a different package